resources/lang/es-VE/validation.php

```diff
@@ -1,138 +0,0 @@
-<?php
```

Comment:
We don't want to remove these language files - best to just make your additions to the `en` language files and leave the rest alone - they'll get automatically updated as our translators find the new translation strings to translate.

Reply:
Will do the update